streams/src/test/java/org/apache/kafka/streams/integration/EosBetaUpgradeIntegrationTest.java

```diff
@@ -113,14 +114,7 @@
     private static final List<KeyValue<KafkaStreams.State, KafkaStreams.State>> CRASH =
         Collections.unmodifiableList(
             Collections.singletonList(
-                KeyValue.pair(KafkaStreams.State.RUNNING, KafkaStreams.State.ERROR)
-            )
-        );
-    private static final List<KeyValue<KafkaStreams.State, KafkaStreams.State>> CLOSE_CRASHED =
-        Collections.unmodifiableList(
-            Arrays.asList(
-                KeyValue.pair(KafkaStreams.State.ERROR, KafkaStreams.State.PENDING_SHUTDOWN),
-                KeyValue.pair(KafkaStreams.State.PENDING_SHUTDOWN, KafkaStreams.State.NOT_RUNNING)
+                KeyValue.pair(State.PENDING_ERROR, State.ERROR)
```

Comment:
Using just `PENDING_ERROR -> ERROR` because the transition to `PENDING_ERROR` can be from multiple sources. Also that transition is already tested so this check implies it